src/nifake/metadata/config.py

```diff
@@ -22,5 +22,8 @@
         'abort_function': 'Abort',
     },
     'init_function': 'InitWithOptions',
+    'custom_types': [
+        {'file_name': 'custom_struct', 'python_name': 'custom_struct', 'ctypes_name': 'CustomStruct', },
```

Comment:
I think more intuitive from a client point of view would be to have:
    
    {'file_name': 'custom_struct', 'python_name': 'CustomStruct', 'ctypes_name': '_CustomStruct', },

Reply:
The 'ctypes_name' (now 'ctypes_type') needs to match what is in the API definition (function.py). For NI-SCOPE this will be `wfm_info` which is not a name that will make flake8 happy.

See comment above